evm/vm/base.py

```diff
@@ -218,10 +232,7 @@ def state_in_temp_block(self):
         header = self.block.header
         temp_block = self.generate_block_from_parent_header_and_coinbase(header, header.coinbase)
         prev_hashes = (header.hash, ) + self.previous_hashes
-        gas_used = 0
-        if self.receipts:
-            gas_used = self.receipts[-1].gas_used
-        state = self.get_state(self.chaindb, temp_block, prev_hashes, gas_used)
+        state = self.get_state(self.chaindb, temp_block, prev_hashes, gas_used=0)
```

Comment:
Hah, yay for unrelated bugfixes (`gas_used` should always start at 0 in a temp block).